### DISCUSSION TITLE
ReInstall outLine Manager

### DISCUSSION THREAD

**Discussion by alexsandrskopin** [OutlineFoundation/outline-sdk#191] [Q&A] [answered]
I reinstalled outline manager, how can I get login information (to connect to my server)? I don't want to reinstall the entire server. I just want to connect the manager (how to get the magic green line) which appears after installing the server

**Comment by sbruens** [comment, marked as answer]
Moving this to the Outline Manager repo: https://github.com/Jigsaw-Code/outline-apps/issues/1923
